app/src/main/java/com/conupods/OutdoorMaps/Models/PointsOfInterest/Photo.java

```diff
@@ -2,20 +2,20 @@
 
 public class Photo
 {
-    private String photo_reference;
+    private String photoReference;
```

Comment:
Do not change the name of this attribute, it must match the attribute name that is within the JSON response object. Revert change.